apps/web/src/routes/(marketing)/(auth)/register/+page.server.ts

```diff
@@ -93,8 +93,8 @@ export const actions: Actions = {
 			// .insert([{ display_name: body.display_name as string }]);
 
 			if (profileError) {
-				console.error("Failed to create user profile:", profileError);
-				return fail(500, { error: "Failed to create user profile" });
+				console.error("email already registered!", profileError);
+				return fail(500, { error: "email already registered!" });
```

Comment:
here it's just checking for any profile error, the fail response should match the actual error. This seems to just be saying email is already registered for all errors